assignments/array-methods.js

```diff
@@ -56,28 +56,74 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+function totalName (x) {fullName.push(`${x.first_name}` + ' ' + `${x.last_name}`);}
```

Comment:
If you're using backticks you don't need to separate it with the plus signs. You can simply add a space.